enrollment-server/src/main/java/com/wultra/app/docverify/zenid/service/ZenidRestApiService.java

```diff
@@ -184,6 +181,18 @@ public String getFilename() {
         return restTemplate.exchange(apiPath, HttpMethod.GET, entity, ZenidWebInvestigateResponse.class);
     }
 
+    /**
+     * Initializes the SDK of ZenID
+     *
+     * @param token Initialization token
+     * @return Response entity with the SDK initialization result
+     */
+    public ResponseEntity<ZenidWebInitSdkResponse> initSdk(String token) {
+        String apiPath = String.format("/api/initSdk?token=%s", token);
```

Comment:
UriVariables may be passed directly to the method `exchange(String url, HttpMethod method, @Nullable HttpEntity<?> requestEntity, Class<T> responseType, Object... uriVariables)`

Reply:
Nice tip! But I will not include it here as there would be an inconsistent approach as the building of full url path is on all other places already. We could apply it later during refactor to our https://github.com/wultra/lime-java-core